README.md

```diff
@@ -417,9 +442,18 @@ To play around with the language in your browser, you can go to the [playground]
 To use the language locally, follow the following steps:
 
 1. `git clone` the repo.
-2. within the `ok` directory run `go install`.
+2. Within the `ok` directory run `go install`
 3. Run `ok` without any arguments to bring up the REPL, or you can run an _OK?_ file with `ok test.ok`.
 
+
+Note: 
```

Comment:
If you were in the ok directory (as opposed to the root directory) then `go install` should have worked.